pkg/run/run.go

```diff
@@ -218,18 +262,13 @@ func (r *RunOptions) PreRunCheck(kclient kubernetes.Interface) error {
 
 	privileged := &rbacv1.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: pkg.PrivilegedClusterRoleBinding,
+			Name: "system:openshift:scc:privileged",
 		},
 		Subjects: []rbacv1.Subject{
 			{
-				Kind:     rbacv1.GroupKind,
-				APIGroup: rbacv1.GroupName,
-				Name:     "system:authenticated",
-			},
-			{
-				Kind:     rbacv1.GroupKind,
+				Kind:     rbacv1.ServiceAccountKind,
 				APIGroup: rbacv1.GroupName,
```

Comment:
Since ServiceAccounts are core group, I think this should be `""` or just unspecified.